wgpu-custom/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "wgpu-custom"
```

Comment:
What is the purpose of this crate?

Reply:
Example how to use custom context.

Reply:
Can this be either an example or a test? We don't really want more crates unless there's a strong motivating reason.

Reply:
This should actually be example crate like those in [examples/standalone](https://github.com/gfx-rs/wgpu/tree/trunk/examples/standalone), but at the time of writing this PR, that didn't exists.

Reply:
Done in https://github.com/gfx-rs/wgpu/pull/6658/commits/52d897ea6a84165f66a217675abbc9586fd66b48, I also write a description for crate.